### ISSUE TITLE
Prevent instant death not working correctly/affected by "extended armor" setting 

### ISSUE THREAD

**Issue by Felixakos** [acemod/ACE3#1848]
ACE 3 Version: 3.1.1
CBA Version:    1.1.23.150604

Mods:
@cba_a3
@ace3

Description:
The "Prevent instant death" setting in the medical module is affected by the "extended armor" setting. 
With "extended armor" on, a grenade directly to your feet will not kill you and only move you into unconscious state (as intended). With "extended armor" off, the same grenade will kill you. Same behavior applies to bullets, etc. Enemies will sometimes directly kill you if "extended armor" is set to off. 

Probably has something to do with the damage multiplier. Simply increasing the amount of damage required to directly kill a solider (but keeping the amount require to move a unit into unconscious the same) should fix it.

Current work around is increasing the "player damage" setting in the medical module. 1.2 is already enough to prevent a player from instantly dying trough a grenade.

**Comment by alganthe** [comment]
Nope the advanced armor change the way damage is handled and thus may result in different behavior with grenades and bullets.

It pretty much decrease the damage inflicted upon the unit.

**Comment by Ligthert** [comment]
I am not using extended armor and still Prevent Instant Death doesn't prevent instant death. And this is with ACE 3 v3.1.1

**Comment by alganthe** [comment]
Prevent death put a unit unconscious if the amount of damage should've killed it, the damage modifiers doesn't apply here, reproduction steps / mission would be very helpful here.

**Comment by Felixakos** [comment]
Alright, so it seems the medical settings module must be combined with the "revive" module, in order for prevent insta death to work..

I'll keep this issue open for it, since it require either better documentation or the prevent isnta death setting should work on its own

**Comment by Glowbal** [comment]
Please confirm this is still an issue with ACE 3.2.1

**Comment by Ligthert** [comment]
I cannot confirm this when used with RHS without the optional compat PBOs

During last night's op prevent Instant Death was set to on and player damage was set to 1.5.

When the RHS BTR came around the corner it rinsed half my section. :s ( Video from a teammate: https://www.youtube.com/watch?v=xNx5vp_X3HI&t=3h47m28s )

**Comment by Felixakos** [comment]
I can confirm it's still an issue with ACE 3.2.1

Test setup is simple: 
Place m112 demolition block at your feet, detonate it, die. Same happens with grenades.

Once you place the revive module, prevent insta-death works. 

Repo missions:
Only with the medical module
https://www.dropbox.com/s/1qp7b9c9k1a2ddt/ACE3_med.7z?dl=0

With medical module + revive module
https://www.dropbox.com/s/c60zb9ltzx3hzyi/ACE3_med_with_revive.7z?dl=0

Important: Needs to be in MP environment

Mods:
ACE       3.2.1
CBA_A3 1.1.23.150604

**Comment by brokenhands** [comment]
I ran into this issue and believe it's actually related to the medical module not being (fully?) activated. I came to this conclusion when I increased the player damage to 1.2 intially, and then 4.0 eventually; with no luck whatsoever. The units all seemed to be made of glass. The map I was working on uses ACE/ALiVE/RHS/LeightsOPFOR, so it's a big ball of wax. I tried reducing the mods, had the same issue. I tried removing the scripted components, same issue. I tried removing all the ACE modules that I wasn't using, same issue. 

I then went into the editor, which had a ton (100s) of placed objects across Altis. Lots of them had init scripts. I removed individual placed objects with little success. I then ended up removing a set of waypoints for some generated units, and blam: the medical system suddenly works as expected; revive and all. Went back to the base map that I'd started with, removed just those waypoints: works as intended.

It's worth noting that there are other waypoints and similar units elsewhere on the map that didn't cause this issue. The only obvious difference was that the removed ones were placed near the ACE modules.
